src/main/java/com/coveo/saml/ValidatorUtils.java

```diff
@@ -248,6 +257,11 @@ public static void validate(
     validateSignature(response, credentials);
   }
 
+  public static void validateIdpMetadata(InputStream metadata) throws SamlException, XMLParserException {
```

Comment:
this does not seem used, which mean the `createDOMParser` could be moved back to its original location.

Reply:
I only wanted to create a possibility to validate Idp metadata in the library and to maintain this possibility here instead of implementing it externally myself.

If you want to remove it, it's okay for me.

Reply:
At this point, I'd like to keep the public api as it was so yes, please remove it.